### PR TITLE
Skip existing files&dirs in env/initialize.sh

### PR DESCRIPTION
**Pull request by emaxx-google**
Change the env/initialize.sh script to be idempotent by default, so that
already existing tools and libraries are left intact by it. Add the "-f"
option that allows to revert to the previous "destructive" behavior.

This is a preparation change for introducing the WebAssembly build tools
initialization in #177, since it'll make it possible for developers to
rerun "initialize.sh" in order to get WebAssembly tools without
destroying their existing Native Client related state.

### CODE REVIEW

File: env/initialize.sh [hunk 1]
```diff
@@ -14,9 +14,18 @@
...
+# all necessary tools and libraries.
+#
+# By default, existing directories and files are left intact (except the
+# "./activate" file), so it's safe to run this script multiple times. Add the
+# "-f" flag in order to enforce overwriting the existing state.
+
 
 set -eu
 
+is_forced=0
```
Comment (Fabian-Sommer) [inline env/initialize.sh:27]: nit: This name doesn't really convey what is enforced - maybe rename it to force_reinitialization or so

File: env/initialize.sh [hunk 2]
```diff
@@ -31,6 +40,10 @@ log_error_message() {
 }
 
 initialize_depot_tools() {
+  if [ -d ./depot_tools -a "${is_forced}" -eq "0" ]; then
```
Comment (Fabian-Sommer) [inline env/initialize.sh:43]: Not familiar with the syntax here - I assume this checks 1) existence of the folder `./depot_tools` and 2) is_forced being equal to 0 - right?

**Review by Fabian-Sommer — APPROVED** [review]
LGTM % nit

File: env/initialize.sh (hunk 1 above)
Comment (emaxx-google) [reply env/initialize.sh:27]: Done

File: env/initialize.sh (hunk 2 above)
Comment (emaxx-google) [reply env/initialize.sh:43]: Correct - see, e.g., https://tldp.org/LDP/abs/html/comparison-ops.html .

**Pull request merged**